docs/sources/flow/reference/components/prometheus.exporter.azure.md

```diff
@@ -0,0 +1,147 @@
+
+## Authentication
+
+Grafana agent must be running in an environment with access to Azure. The exporter uses the Azure SDK for go and supports [authentication](https://learn.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication?tabs=bash#2-authenticate-with-azure).
+
+The account used by Grafana Agent needs:
+
+- [Read access to the resources that will be queried by Resource Graph](https://learn.microsoft.com/en-us/azure/governance/resource-graph/overview#permissions-in-azure-resource-graph)
+- Permissions to call the [Microsoft.Insights Metrics API](https://learn.microsoft.com/en-us/rest/api/monitor/metrics/list) which should be the `Microsoft.Insights/Metrics/Read` permission
+
+## Usage
+
+```river
+prometheus.exporter.azure LABEL {
+        subscriptions = [
+                SUB_ID_1,
+                SUB_ID_2,
+                ...
+        ]
+
+        resource_type = RESOURCE_TYPE
+
+        metrics = [
+                METRIC_1,
+                METRIC_2,
+                ...
+        ]
+}
+```
+
+## Arguments
+
+You can use the following arguments to configure the exporter's behavior.
+Omitted fields take their default values.
+
+| Name                          | Type           | Description                                                          | Default                                                                       | Required |
+| ----------------------------- | -------------- | -------------------------------------------------------------------- | ----------------------------------------------------------------------------- | -------- |
+| `subscriptions`               | `list(string)` | List of subscriptions to scrap metrics from.                         |                                                                               | yes      |
+| `resource_type`               | `string`       | The Azure Resource Type to scrape metrics for.                       |                                                                               | yes      |
+| `metrics`                     | `list(string)` | The metrics to scrape from resources.                                |                                                                               | yes      |
+| `resource_graph_query_filter` | `string`       | The [Kusto query][] filter to apply when searching for resources.    |                                                                               | no       |
+| `metric_aggregations`         | `list(string)` | Aggregations to apply for the metrics produced.                      |                                                                               | no       |
+| `timespan`                    | `string`       | [ISO8601 Duration][] over which the metrics are being queried.       | `"PT1M"`                                                                      | no       |
```

Comment:
Could we explain what `"PT1M"` means and provide a couple of examples the users could set this to?

Reply:
I think adding a quick explanation below that `PT1M` means it defaults to every one minute (right? 😅) wouldn't hurt.